kaggle-classification/trainer/model.py

```diff
@@ -42,15 +42,13 @@
 MAX_LABEL = 2
 MAX_DOCUMENT_LENGTH = 500 # Max length of each comment in words
 
+# CNN parameters
 CNNParams = namedtuple(
-  'CNNParams', ['WINDOW_SIZE', 'EMBEDDING_SIZE','POOLING_WINDOW', 'POOLING_STRIDE',
-                'N_FILTERS', 'FILTER_SHAPE1', 'FILTER_SHAPE2'])
-cnn_values = {'WINDOW_SIZE':20, 'EMBEDDING_SIZE':20, 'POOLING_WINDOW':4,
-              'POOLING_STRIDE':2, 'N_FILTERS':10}
-cnn_values['FILTER_SHAPE1'] = [cnn_values['WINDOW_SIZE'], cnn_values['EMBEDDING_SIZE']]
-cnn_values['FILTER_SHAPE2'] = [cnn_values['WINDOW_SIZE'], cnn_values['N_FILTERS']]
-CNN_PARAMS = CNNParams(**cnn_values)
+  'CNNParams',['EMBEDDING_SIZE','N_FILTERS', 'FILTER_SIZES', 'DROPOUT_KEEP_PROB'])
+CNN_PARAMS = CNNParams(
+  EMBEDDING_SIZE=20, N_FILTERS=10, FILTER_SIZES=[2,3,4,5],  DROPOUT_KEEP_PROB=.75)
```

Comment:
This seems like quite a small embedding size. As you continue to experiment with parameters, I would try adjusting the embedding size up to 50-100 which is more standard.